addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -40,45 +61,81 @@ registerWebsitePreviewTour('edit_link_popover', {
         trigger: FIRST_PARAGRAPH,
         run: "editor Paragraph", // Make sure the selection is set in the paragraph
     },
+    {
+        content: "Click on a paragraph",
+        trigger: FIRST_PARAGRAPH,
+        async run(actions) {
+            const range = document.createRange();
+            const selection = this.anchor.ownerDocument.getSelection();
+            range.selectNodeContents(this.anchor);
+            selection.removeAllRanges();
+            selection.addRange(range);
+            await actions.click();
+        },
+    },
     {
         content: "Click on 'Link' to open Link Dialog",
-        trigger: "#toolbar:not(.oe-floating) #create-link",
+        trigger: ".o-we-toolbar button[name='link']",
         run: "click",
     },
     {
         content: "Type the link URL /contactus",
-        trigger: '#o_link_dialog_url_input',
+        trigger: '.o-we-linkpopover .o_we_href_input_link',
         run: "edit /contactus",
     },
-    ...clickFooter,
+    ...applyLinkAndClickBody,
     {
         content: "Click on newly created link",
         trigger: `${FIRST_PARAGRAPH} a`,
-        run: "click",
+        async run(helpers) {
+            await helpers.click();
+            const el = this.anchor;
+            const sel = el.ownerDocument.getSelection();
+            sel.collapse(el.childNodes[1], 1);
+            el.focus();
+        },
```

Comment:
We can use utils method implemented in https://github.com/odoo-dev/odoo/pull/4590